cmd/parse.go

```diff
@@ -76,6 +123,8 @@ func parse(args []string, stdout io.Writer, stderr io.Writer) int {
 }
 
 func init() {
-	parseCommand.Flags().VarP(parseParams.format, "format", "f", "set output format")
+	parseCommand.Flags().VarP(configuredParseParams.format, "format", "f", "set output format")
+	parseCommand.Flags().StringVarP(&configuredParseParams.jsonInclude, "json-include", "", "", "select optional elements, current options: locations, comments. E.g. --json-include locations,-comments will include locations and exclude comments.")
```

Comment:
It might be good to state the default here. Comments are included today by default, right? While location will require this flag to be set to include `locations`... correct?

Reply:
👍 I have added this in 876c1eb002c29752cfeb1f38ac3b0207a368bcb2